Sources/Networking/Combine/APIService+Combine.swift

```diff
@@ -27,7 +27,7 @@ public extension APIServiceable {
                 decoder: decoder,
                 router: router,
                 session: session,
-                completion: { promise($0.toCombineResult) }
+                completion: { promise($0) }
```

Comment:
This should also compile 😄 

```suggestion
                completion: promise
```

Reply:
Yup, it does. Nice 🙌